pom.xml

```diff
@@ -192,6 +192,27 @@
                 <surefire.options>--add-exports java.base/java.lang=ALL-UNNAMED --add-exports java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED</surefire.options>
             </properties>
         </profile>
+        <!-- Checkstyle 10+ requires Java 11+ -->
+        <profile>
+            <id>checkstyle-jdk11+</id>
+            <activation>
+                <jdk>(,11)</jdk>
```

Comment:
Is the syntax right here? Reading https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html implies it should be something like `[11,)`

Reply:
Oh never mind, I see that the logic is inverted

Reply:
Correct, it's a little awkward, but I thought preferable to "skip" with the activated profile rather than move all the checkstyle config inside it.